### PR TITLE
SP-NONE: Add support for EL messages. Fix LEC and ELT

### PR DESCRIPTION
**Pull request by michalerazmus-silvair**
This PR adds support for Silvair Emergency Lighting messages. It also fixes some issues with Property IDs in Light Extended Controller and Emergency Lighting Test.

### CODE REVIEW

File: tests/test_capnproto.py [hunk 1]
```diff
@@ -270,6 +270,25 @@
...
+    bytes.fromhex("EA360100"),
+    bytes.fromhex("EA360101"),
+    bytes.fromhex("EA360102"),
+    bytes.fromhex("EA360103"),
+    bytes.fromhex("EA360104"),
+    bytes.fromhex("EA36010500"),
+    bytes.fromhex("EA36010680FF"),
+    # bytes.fromhex("EA36010780FF3412"),
+    # bytes.fromhex("EA36010880FF3412"),
+    # bytes.fromhex("EA36010980FF3412"),
```
Comment (michalerazmus-silvair) [inline tests/test_capnproto.py:283]: I don't understand why these test cases fail

**Comment by michallowasrzechonek-silvair** [comment]
I'll go through this shortly 😍

**Review by kamiljanowski-silvair — APPROVED** [review]
I tried to find something invalid, but I didn't xD 

Good job :+1:

**Comment by michalerazmus-silvair** [comment]
@michallowasrzechonek-silvair good point. I included refactoring into commits. I also checked and every commit is properly formatted and has passing all tests.

**Review by michallowasrzechonek-silvair — APPROVED** [review]

**Comment by michallowasrzechonek-silvair** [comment]
Great work, thanks!

**Pull request merged**